tests/unitarize_link_test.cpp

```diff
@@ -36,7 +36,7 @@ static double unitarize_eps  = 1e-6;
 static bool reunit_allow_svd = true;
 static bool reunit_svd_only  = false;
 static double svd_rel_error  = 1e-4;
-static double svd_abs_error  = 1e-5;
+static double svd_abs_error  = 1e-4;
```

Comment:
Is this required to make the test pass? Did it pass with the stricter error before the optimization?